recipes/app2.rb

```diff
@@ -192,7 +192,6 @@
     user 'replicant'
     working_directory '/home/replicant/redmine'
     pid_file '/home/replicant/pids/unicorn.pid'
-    exec_start '/home/replicant/.rvm/bin/rvm 2.3.0 do bundle exec '\
-      'unicorn -l 8090 -c unicorn.rb -E deployment -D'
+    exec_start 'RAILS_ENV=production ./script/server -b 140.211.9.86 -p 8090'
```

Comment:
Right, choose whatever version you need but you need to go through rvm regardless.